app/code/Magento/Review/Api/ReviewInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace Magento\Review\Api;
+
+/**
+ * Interface ReviewInterface
+ * @api
+ */
+interface ReviewInterface
+{
+    /**
+     * Return Added review item.
+     *
+     * @param int $productId
+     * @return array
+     *
+     */
+    public function getReviewsList($productId);
+
+    /**
+     * Added review item.
+     * @param int $productId
+     * @param string $title
+     * @param string $nickname
+     * @param string $detail
+     * @param int $ratingValue
+     * @param int $customerId
+     * @param int $storeId
+     * @return boolean
+     *
+     */
+    public function writeReviews(
```

Comment:
I think these are a lot of arguments for a method, is there anyway we can use parameter object or inject dependencies that can retrieve these values or maybe have ReviewInterface have these values as instance properties